modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc

```diff
@@ -0,0 +1,22 @@
+= Zuora - XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== XML Namespace
+
+<No information>
```

Comment:
Put these statements in a source block for the XML Namespace section:

http://www.mulesoft.org/schema/mule/zuora-connector
http://www.mulesoft.org/schema/mule/zuora-connector/current/mule-zuora-connector.xsd

Reply:
These URLs are not found when I put them into the browser.

Reply:
OK